kuksa-sdk/src/main/kotlin/org/eclipse/kuksa/DataBrokerConnection.kt

```diff
@@ -283,6 +277,26 @@ class DataBrokerConnection internal constructor(
         }
     }
 
+    /**
+     * Only a [VssProperty] can be updated because they have an actual value. When provided with any parent
+     * [VssSpecification] then this [update] method will find all [VssProperty] children and updates them instead.
+     * Compared to [update] with only one [Property] and [Datapoint], here multiple [SetResponse] will be returned
+     * because a [VssSpecification] may consists of multiple values which may need to be updated.
+     *
+     *  @throws DataBrokerException in case the connection to the DataBroker is no longer active
+     */
+    suspend fun update(vssSpecification: VssSpecification): List<SetResponse> {
+        val responses = mutableListOf<SetResponse>()
+
+        val propertyDataPointPairs = vssSpecification.createPropertyDataPoints()
```

Comment:
Method is missing transitive throws declaration from DataPointExtension -> VssProperty.datapoint

Reply:
Removed throw and added the fallback to string.